tests/mocha/block_test.js

```diff
@@ -917,50 +898,48 @@ suite('Blocks', function() {
       });
     });
     suite('Remove Connections Programmatically', function() {
+      setup() {
+        this.makeBlock = function(type) {
+          var block = this.workspace.newBlock(type);
+          block.initSvg();
+          block.render();
+          return block;
```

Comment:
This would be nice as a shared helper function, maybe rename to `makeRenderedBlock` and add to shared helpers